activities/Abecedarium.activity/util.js

```diff
@@ -195,26 +199,3 @@ Abcd.mix = function(chain) {
 
 	return mixedchain;
 };
-
```

Comment:
Warning: this code was added by another PR, your source code is probably not the latest one.